### PR TITLE
Ability to set default selection for popup location picker and fix cookie set os-30

### PR DESCRIPTION
**Pull request by dbuzinov-yx**
Provide a way to stay informed of Y events happening in some area.

## Steps for review

- Install from profile or update existing site
- Visit /schedules page
 In case no cookie "openy_preferred_branch" not set should display popup with 'all' radio checked 
- Login to system 
-  Visit /admin/config/content/location-popup
- Should be able to select default location popup
- Save config
- Visit /schedules page
 In case no cookie "openy_preferred_branch" not set should display popup with saved radio checked
-  Click on "Set Location" button
- cookie "openy_preferred_branch" should be set
- Visit /schedules page
Popup should not be displayed. 

Please review, thanks

### CODE REVIEW

**Comment by gundevel** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by podarok** [comment]
ok to test

**Review by podarok — APPROVED** [review]

File: modules/custom/openy_popups/config/schema/openy_popups.schema.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-openy_popups.config:
```
Comment (podarok) [inline modules/custom/openy_popups/config/schema/openy_popups.schema.yml:1]: We need to update a config for the old users. Please update it in hook_update_N

**Review by podarok — CHANGES_REQUESTED** [review]

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build303/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build303/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build303/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build303/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build303/blogWCAG2AAhtmlcs.txt

File: modules/custom/openy_popups/config/schema/openy_popups.schema.yml (hunk 1 above)
Comment (dbuzinov-yx) [reply modules/custom/openy_popups/config/schema/openy_popups.schema.yml:1]: Hello, 
Configuration form submit handler previously used openy_popups.settings

openy/modules/custom/openy_popups/src/Form/SettingsForm.php line 71

$config = \Drupal::service('config.factory')->getEditable('openy_popups.settings');

So only openy_popups.settings store actual setting data. Maybe it was just misspelling. 

I will update hook update to remove not used conf object

**Comment by podarok** [comment]
ok to test

**Comment by podarok** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build310 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build310 |
| Installation process | http://install.openy-dev.ffwua.com/build310/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/310/

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build310/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build310/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build310/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build310/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build310/blogWCAG2AAhtmlcs.txt

**Comment by podarok** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build341 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build341 |
| Installation process | http://install.openy-dev.ffwua.com/build341/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/341/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build341/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build341/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build341/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build341/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build341/blogWCAG2AAhtmlcs.txt

**Pull request merged**